ADBench/plot_graphs.py

```diff
@@ -56,34 +58,54 @@
     "Zygote": ("y", "v", "Julia, Zygote")
 }
 
+# File with input data if specified
+if use_file:
+    arg_idx = sys.argv.index("--use-file") + 1
+    input_file_path = sys.argv[arg_idx]
+
 # Folders
 adbench_dir = os.path.dirname(os.path.realpath(__file__))
 ad_root_dir = os.path.dirname(adbench_dir)
-in_dir = os.path.join(ad_root_dir, "tmp")
+in_dir = os.path.join(ad_root_dir, "tmp") if not use_file else os.path.dirname(input_file_path)
```

Comment:
If `use_file` is false then `in_dir` should never be used, is that not the case?  If it is the case then I would suggest setting `in_dir` to `None` to indicate that (pending a future refactoring where we only ever assign `in_dir` when we know it will be used).